src/System.Data.SQLite/NativeMethods.cs

```diff
@@ -127,6 +127,56 @@ internal static class NativeMethods
 		[DllImport(c_dllName, CallingConvention = c_callingConvention)]
 		public static extern int sqlite3_trace_v2(SqliteDatabaseHandle db, SQLiteTraceEvents eventsMask, SQLiteTraceV2Callback callback, IntPtr userData);
 
+#if NET5_0
+		static NativeMethods()
+		{
+			var assembly = typeof(NativeMethods).Assembly;
+			var configFile = System.IO.Path.GetFileName(assembly.Location) + ".config";
+			if (System.IO.File.Exists(configFile))
+			{
+				try
+				{
+					using var stream = new System.IO.FileStream(configFile, System.IO.FileMode.Open);
+					using var reader = System.Xml.XmlReader.Create(stream);
+					var dllMapConfig = System.Xml.Linq.XDocument.Load(reader);
```

Comment:
```suggestion
					var dllMapConfig = XDocument.Load(configFile);
```

Can we just use the file path overload?

Reply:
I don't see why not, VS code complete didn't offer that one for me.